docs/conf.py

```diff
@@ -48,16 +48,16 @@
 
 # General information about the project.
 project = u'libCellML'
-copyright = u'2014, David Nickerson'
+copyright = u'2014-2015, University of Auckland'
```

Comment:
I have done something similar to "Copyright libCellML Contributors" for OpenCOR. See [here](https://github.com/opencor/opencor/blob/master/NOTICE.txt).